query-engine/query-engine/src/opt.rs

```diff
@@ -107,6 +107,10 @@ pub struct PrismaOpt {
     #[structopt(long, default_value)]
     pub open_telemetry_endpoint: String,
 
+    /// JSON object for tracing headers such as the traceparent
+    #[structopt(long, default_value = "{}")]
+    pub tracing_headers: String,
```

Comment:
Needed for the Client to pass the `traceparent` in `binary` mode.

Reply:
https://github.com/prisma/prisma/pull/14638/files#diff-c9ca2fbbe7015edf91d3a2b3958c74c4fbe5bb61049f70973a0e6a983413c6c8R579